modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,146 @@
+
+[discrete]
+=== View Spaces as List
+
+Previously, viewing your spaces was limited to a tiles format. You can now switch to a list view that presents your projects in a clean, streamlined manner. This new view offers an overview of all your spaces, allowing for easy navigation and a concise visual of your work environment.
+
+This list view provides additional information as name of the:
+
+* manager
+* ammount of members
+* total quota
+* used quota
+* remainig quota
+* status
+* last modfied date
+
+in a more compact and organized manner. This is particularly useful for users who manage multiple spaces, as it allows for efficient scanning and selection. https://github.com/owncloud/web/pull/9195[#9195]
+
+[discrete]
+=== Other Notable Changes
+
+* Bugfix - Fixes ocis postprocessing restart command: Restarts complete postprocessing pipeline if there is no active postprocessing https://github.com/owncloud/ocis/pull/6753[#6753]
+* Enhancement - Add `ocis decomposedfs check-treesize` maintenance command: Verify the treesize metadata of a Space in decomposedfs: https://github.com/owncloud/ocis/pull/6556[#6556]
+* Enhancement - Add Server-sent events Endpoint for Notifications: https://github.com/owncloud/ocis/pull/5998[#5998]
+* Enhancement - Use reva client selectors to improve performance and scalability: https://github.com/owncloud/ocis/pull/6452[#6452]
+* Enhancement - Allow disabling wopi chat to disable the inline chat in ONly Office: https://github.com/owncloud/ocis/pull/6544[#6544]
+* Enhancement - Download a whole Space as Archive: https://github.com/owncloud/web/issues/9056[#9056]
+* Enhancement - We've implemented a new solution to deal with long breadcrumbs even with long folder names: https://github.com/owncloud/web/pull/8984[#8984]
+* Enhancement - Respect archiver limits. The archiver service announces a limit for the accumulated filesize of the currently selected resources. The web UI now respects those limits and shows a disabled download button once the limit has been reached. https://github.com/owncloud/web/pull/9055[#9055]
+* Enhancement - Privacy statement in account menu. We've added the option to add an imprint and privacy statement via the config:
+ https://github.com/owncloud/web/issues/9174[#9174]
+* Enhancement - Allow local storage for auth token. We've introduced a new env var `WEB_OPTION_TOKEN_STORAGE_LOCAL`, when set to true (default), the auth token will be stored in the browser's local storage instead the session storage, this will effect in a persisted login state across multiple browser tabs. https://github.com/owncloud/web/pull/9386[#9386]
+* Enhancement - Add login button to top bar. We've added a login button to the top bar, this might be handy if a user receives a public link, and they want to login with their user account. https://github.com/owncloud/web/pull/9178[#9178]
```

Comment:
```suggestion
* Enhancement - Add login button to top bar. We've added a login button to the top bar, this might be handy if a user receives a public link, and they want to login with their user account: https://github.com/owncloud/web/pull/9178[#9178]
```